include/api.php

```diff
@@ -1576,7 +1577,7 @@ function api_conversation_show($type){
 
 		$a = get_app();
 
-		if (api_user()===false) throw new ForbiddenException();
+		if (api_user()===false) throw new HTTP\ForbiddenException();
```

Comment:
Standards: Please add braces to this conditional statement and spaces around the `===` operator.